cmake/tribol-config.cmake.in

```diff
@@ -28,15 +28,17 @@ if(NOT TRIBOL_FOUND)
   # Language features
   set(TRIBOL_USE_FORTRAN          "@ENABLE_FORTRAN@")
   set(TRIBOL_USE_MPI              "@ENABLE_MPI@")
-  #set(TRIBOL_USE_CUDA            "@TRIBOL_USE_CUDA@")
-  #set(TRIBOL_USE_OPENMP          "@TRIBOL_USE_OPENMP@")
+  set(TRIBOL_USE_CUDA             "@TRIBOL_USE_CUDA@")
+  set(TRIBOL_USE_HIP              "@TRIBOL_USE_HIP@")
+  set(TRIBOL_USE_OPENMP           "@TRIBOL_USE_OPENMP@")
 
   # Config options
   set(TRIBOL_USE_SINGLE_PRECISION "@TRIBOL_USE_SINGLE_PRECISION@")
```

Comment:
Do we want to leave this in here even though we don't support single precision? It would be caught elsewhere?

Reply:
I think there are more changes needed re: single precision hooks in Tribol.  Let's defer changes to a future PR that deals with single precision more thoroughly.